router/router.go

```diff
@@ -107,7 +115,12 @@ func (r *chiWrapper) Method(method, pattern string, h APIHandler) {
 
 // Get adds a GET route
 func (r *chiWrapper) Get(pattern string, fn APIHandler) {
-	r.chi.Get(pattern, r.traceRequest(http.MethodGet, pattern, fn))
+
+	wrapper := new(chiWrapper)
+	*wrapper = *r
+	wrapper.tracingPrefix = sanitizePattern(pattern)
+
+	wrapper.chi.Get(pattern, wrapper.traceRequest(http.MethodGet, pattern, fn))
```

Comment:
this won't work because the `tracingPrefix` is associated with the root `chi` router. Though I'm not totally convinced that I can't get this to work...just couldn't at the time.